### ISSUE TITLE
Problem with signing utility.batch()

### ISSUE THREAD

**Issue by stepanLav** [Zondax/ledger-kusama#180]
## Summary

In [NovaWallet](https://novawallet.io/) we encountered a problem when trying to sign `utility.batch()` and `utility.batchAll()` which contains nested calls, [example of transaction](https://kusama.subscan.io/extrinsic/9060589-2). When we try to sign such  transaction with Ledger Nano X, an error occurs.

## Steps to reproduce
1. Build utility.batchAll() transaction with few nested calls, [example](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/extrinsics/decode/0x180208060000eaa4f8a3b4e16e3ea6718b4263378749913a2d607e9475953ab8d906bf9f741202093d000006050400eaa4f8a3b4e16e3ea6718b4263378749913a2d607e9475953ab8d906bf9f7412)
2. Send that transaction to Ledger Nano X for sign

### Actual result:
The Ledger return code - **`0x6D00`**

<img width="400" alt="Screenshot 2022-12-09 at 10 09 41" src="https://user-images.githubusercontent.com/40560660/206645474-202d52fc-dc43-4d3a-b65b-be6ab41026ca.png">

## Environment
Ledger Nano X
App version: Kusama 17.9340

:link: [zboto Link](https://app.clickup.com/t/39qqzhn)

**Comment by carlosala** [comment]
That is not reproducible to me. I was able to sign that same transaction in my Ledger device (using polkadot.js) without problem. That error you get, INS not supported, is not related to which transaction you are trying to sign, it's related to APDU protocol. Please, check that you are building the APDU message properly.

**Comment by stepanLav** [comment]
> That is not reproducible to me. I was able to sign that same transaction in my Ledger device (using polkadot.js) without problem. That error you get, INS not supported, is not related to which transaction you are trying to sign, it's related to APDU protocol. Please, check that you are building the APDU message properly.

Yes, I double-checked it and it works! It looks like I tried it on an old application. Currently almost all operations with stake are work properly, including: Unstake, Rebond, Bond more, Change validators, change payout account, Payout rewards, set controller, which is great!
But I found a special case for our workflow where it is not working. If I want to remove all my Stake, then our app will send an utility.batchAll() operation with staking.chill() and staking.unbound() calls. But for some reason Ledger can't sign it. [Example](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama-rpc.polkadot.io#/extrinsics/decode/0x18020806060602025a6202)
<img width="200" alt="Screenshot 2022-12-15 at 10 18 26" src="https://user-images.githubusercontent.com/40560660/207797696-9389d276-436f-4b0a-954c-6739e024ef43.png">
